File: src/main/java/no/nav/domain/pensjon/regler/grunnlag/RegulerBeregningGrunnlag.java
package no.nav.domain.pensjon.regler.grunnlag;

import no.nav.domain.pensjon.regler.Pakkseddel;
import no.nav.domain.pensjon.regler.beregning.Beregning;
import no.nav.domain.pensjon.regler.vedtak.VilkarsVedtak;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.Date;
import java.util.List;

public class RegulerBeregningGrunnlag implements Serializable {
    private Beregning beregning1967;
    private Date virkFom;
    private List<Uttaksgrad> uttaksgradListe = new ArrayList<Uttaksgrad>();
    private List<VilkarsVedtak> brukersVilkarsvedtakListe = new ArrayList<VilkarsVedtak>();
    private Persongrunnlag sokersPersongrunnlag;
    private Persongrunnlag epsPersongrunnlag;
    private Pakkseddel pakkseddel;

}

File: src/main/kotlin/no/nav/pensjon/regler/domain/grunnlag/RegulerBeregningGrunnlag.kt
package no.nav.pensjon.regler.domain.grunnlag

import no.nav.pensjon.regler.domain.Pakkseddel
import no.nav.pensjon.regler.domain.beregning.Beregning
import no.nav.pensjon.regler.domain.vedtak.VilkarsVedtak
import java.io.Serializable
import java.util.*
import java.time.LocalDate

class RegulerBeregningGrunnlag : Serializable {
    var beregning1967: Beregning? = null
    @Deprecated("Use virkFomLd instead")
    var virkFom: Date? = null
    var virkFomLd: LocalDate? = null
    var uttaksgradListe: List<Uttaksgrad> = mutableListOf()
    var brukersVilkarsvedtakListe: List<VilkarsVedtak> = mutableListOf()
    var sokersPersongrunnlag: Persongrunnlag? = null
    var epsPersongrunnlag: Persongrunnlag? = null
    var pakkseddel: Pakkseddel? = null
}